### DISCUSSION TITLE
Logged mileage in charging stats dashboard

### DISCUSSION THREAD

**Discussion by djpalmis** [teslamate-org/teslamate#2945] [Ideas]
Every time I show my total charging cost to a friend, I need to know the mileage that statistics are based on.
So that's the request: can you please show the distance logged in charging stats dashboard?

**Comment by cwanja** [comment]
What you would like added, if I understand, is the `mi driven` panel from `Drive Stats` added to `Charge Stats` as well?

**Reply by djpalmis** [reply]
Oh.. that's where the information is!
Thank you!
..I confess you that every time I go to the mileage dashboard.. and I find the delta mileage doing the math in my head 😄
Thank you again!
